daemon/src/projection.rs

```diff
@@ -753,10 +753,25 @@ impl From<Order> for CfdOrder {
                 .whole_seconds()
                 .try_into()
                 .expect("settlement_time_interval_hours is always positive number"),
-            // FIXME: Replace dummy values for opening fee, funding rates and time
+            // FIXME: Replace dummy value for opening fee
             opening_fee: Amount::from_sat(123 * 24),
-            funding_rate_annualized_percent: "18.5".to_string(),
-            funding_rate_hourly_percent: "0.002345".to_string(),
+            funding_rate_annualized_percent: order
+                .funding_rate
+                .to_decimal()
+                .checked_mul(Decimal::from(
+                    (24 / order.settlement_interval.whole_hours()) * 365,
+                ))
+                .expect("not to overflow")
```

Comment:
The way we handled errors here previously is to fall back to an `Option` so we can at least display as much as possible and not everything blowing up. Perhaps also worth to create a new-type `AnnualizedFundingRate` and encapsulate this.

Reply:
To make sure that these things don't panic in any case, proptests would be good. If you extract this logic into a function, it should be pretty easy to proptest.

Reply:
Happy to create a newtype, but this cannot blow up - the rate can never be above `1.0`. IMHO If this blew up, we have a serious problem that should not be swept under the rug.

Reply:
as this is a string, I could fallback to either an empty string or `--` would that be a preferred solution for you?

Reply:
> as this is a string, I could fallback to either an empty string or `--` would that be a preferred solution for you?

My preferred solution would be an `Option`. In other places, the frontend knows that some of these are optional and displays a skeleton if the value is not available for some reason.